src/usr/local/www/services_dhcp.php

```diff
@@ -332,7 +332,7 @@ function validate_partial_mac_list($maclist) {
 		$input_errors[] = gettext("If a mac deny list is specified, it must contain only valid partial MAC addresses.");
 	}
 
-	if (($_POST['ntp1'] && (!is_ipaddrv4($_POST['ntp1']) && !is_hostname($_POST['ntp1']))) || ($_POST['ntp2'] && (!is_ipaddrv4($_POST['ntp2']) && !is_hostname($_POST['ntp2'])))) {
+	if (($_POST['ntp1'] && (!is_ipaddrv4($_POST['ntp1']) && !is_hostname($_POST['ntp1']))) || ($_POST['ntp2'] && (!is_ipaddrv4($_POST['ntp2']) && !is_hostname($_POST['ntp2']))) || ($_POST['ntp3'] && (!is_ipaddrv4($_POST['ntp3']) && !is_hostname($_POST['ntp3'])))) {
```

Comment:
Could you split this into multiple lines to make it easier to read? I know the old one was all one line but it was far too long as-is. Indent the line with four spaces after the leading tabs.

See [Indent Style](https://www.netgate.com/docs/pfsense/development/developer-style-guide.html#indent-style) for more info.